content/bn/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,86 @@
+---
+reviewers:
+- lavalamp
```

Comment:
I think comment out will be good for us 

```suggestion
#  reviewers:
#  - lavalamp ( The list of approvers is not necessary for the localized version. However, it is included because it helps maintain a certain line break, which further aids in updating a file.That's why it's kept in comment form. ) 
```